### ISSUE TITLE
Question about Gaussian noise simulation for the tracking labels in the training data generation process

### ISSUE THREAD

**Issue by Uio96** [xingyizhou/CenterTrack#218]
Great work! I was wondering about the logic behind the tracking label generation. 

In your codes (https://github.com/xingyizhou/CenterTrack/blob/master/src/lib/dataset/generic_dataset.py#L230-L241), you mention that when there is no lost disturbance, there will be some Gaussian noise in the heatmap, and the tracking label is assigned with the ground truth location. On the other hand, if there exists a lost disturbance, the tracking label is assigned with the noisy location. 

I do not quite understand this implementation idea.  The tracking prediction is to capture the offset from the position in the current frame to the position in the previous frame. Based on your implementation, you are trying to track the ground truth offset when piror heat in the previous frame is available but track the current location to the center of the prior heatmap input for the previous frame when piror heat is lost. Is the tracking prediction supposed to always track the ground truth offset (no matter prior heat is lost or not) or just from the current location to the center of the prior heatmap input for the previous frame when prior heat is available?

I am not sure if you have already investigated that so there is some advantage of the current implementation. Thank you so much.
